ComSemApp/student/views.py

```diff
@@ -789,16 +790,46 @@ def get_object(self):
         """
         return self.course
 
-
     def get_context_data(self, **kwargs) -> dict[str, dict[str, str | int | float]]:
         """
             implements Django's DetailView get_context_data function 
             Returns:
-                context -- contains values needed for speaking practice result page 
+                context -- context data used by assessment_generator.html
```

Comment:
I'm a bit confused why this context data is being used by assessment generator rather than results when it is in the results view page.

Reply:
I copied and pasted then modified this code, will fix comment